### ISSUE TITLE
Basic interpreter

### ISSUE THREAD

**Issue by ystrem** [nekromant/esp8266-frankenstein#11]
Hi, what do you about implementing Basic interpreter to Frankenstein ?
 https://github.com/BleuLlama/TinyBasicPlus

**Comment by dvv** [comment]
Or https://code.google.com/p/picoc/
Indeed, a scripting capabilities would be a killer feature

**Comment by ystrem** [comment]
looks nice :]

**Comment by nekromant** [comment]
Yeah, picoc looks really interesting, I'll definetely give it a spin

**Comment by dvv** [comment]
And yet one: http://www.compuphase.com/pawn/pawn.htm
TIA

**Comment by dvv** [comment]
Any progress or conclusion here?

**Comment by mmiscool** [comment]
Check out http://esp8266basic.com

I have been working on it for about 2 weeks now. It will certanly be getting evin better over time.

**Comment by ildar** [comment]
Isn't Lua scripting enough? -> NodeMCU
